oshdb-api/src/main/java/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapAggregator.java

```diff
@@ -997,4 +997,68 @@ public X setValue(X value) {
         TreeMap::new
     ));
   }
+  
+  /*
+   * A generic Pair class for holding key/value pairs
+   */
+  public static class Pair<U,V> {
```

Comment:
shouldn't this be in a separate java source file? also, I really liked the separation between mutable and immutable pairs in apache.commons.lang3. Would be nice if we kept that.

Reply:
actually, I would be nice if this has a more meaning full name what describes for what this pair class is used for.
To make this class a nested class instead a own was intended to encourage to not use this class in all cases.
There is a reason why this simple class is not included in the java.lang or even guava
https://github.com/google/guava/wiki/IdeaGraveyard
https://mail.openjdk.java.net/pipermail/core-libs-dev/2010-March/003995.html

But I'm also ok with it to have this in a separatet java source file.

Reply:
actually, after a second look, I realized that this class (and previously, the `MutablePair<>`s used in this class) is only used to store pairs of values (generic type `X`) that are associated with an index (generic type `U`) in an internal `MapReducer` object (see above in https://github.com/GIScience/oshdb/pull/79/files#diff-5c05e36796aa82dfa133e2fa41932bb2R87). So, this class is totally fine to live here and can even be made private